app/components/podcasts.js

```diff
@@ -32,11 +31,10 @@ export default class PodcastsSearch extends Component {
   @action
   fetchPodcasts() {
     const query = this.args.query;
-    console.log(query);
     let podcastsPromise = this.store.queryRecord('podcast', query).then((podcast) => {
       return hash({
         tracks: podcast.get('tracks'),
-        meta: App.storeMeta['podcast'],
+        meta: podcast.meta,
```

Comment:
I think the behaviour is unchanged doing it this way, but I can't remember what we were using App.storeMeta for anyway so, this is more readable I suppose.